crates/uv/src/commands/publish.rs

```diff
@@ -0,0 +1,74 @@
+use crate::commands::{human_readable_bytes, ExitStatus};
+use crate::printer::Printer;
+use anyhow::{bail, Result};
+use owo_colors::OwoColorize;
+use std::fmt::Write;
+use tracing::info;
+use url::Url;
+use uv_client::{BaseClientBuilder, Connectivity};
+use uv_configuration::{KeyringProviderType, TrustedHost};
+use uv_publish::{files_for_publishing, upload};
+
+pub(crate) async fn publish(
+    paths: Vec<String>,
+    publish_url: Url,
+    keyring_provider: KeyringProviderType,
+    allow_insecure_host: Vec<TrustedHost>,
+    username: Option<String>,
+    password: Option<String>,
+    connectivity: Connectivity,
+    native_tls: bool,
+    printer: Printer,
+) -> Result<ExitStatus> {
+    if connectivity.is_offline() {
+        bail!("You cannot publish files in offline mode");
```

Comment:
```suggestion
        bail!("Unable to publish files in `--offline` mode");
```

Reply:
Are we always going to stylize with `--`? Can't `offline` have come from the config?